### DISCUSSION TITLE
How to use Vertical tabs

### DISCUSSION THREAD

**Discussion by dbm-git** [carbon-design-system/carbon-components-svelte#1982] [General]
We can't find in the documentation how to use Vertical tabs. Is it possible to use this component with svelte and how ? Thanks

**Comment by metonym** [comment]
`TabsVertical` has not been implemented yet. Issue to track this: https://github.com/carbon-design-system/carbon-components-svelte/issues/2726

**Comment by metonym** [comment]
Vertical tabs are shipped: https://svelte.carbondesignsystem.com/components/Tabs#vertical
